common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4923,4 +4927,20 @@ object CelebornConf extends Logging {
         s"Invalid maxEncryptedBlockSize, must be a position number upto ${Int.MaxValue}")
       .createWithDefaultString("64k")
 
+  val LOG_MASTER_CONF: ConfigEntry[Boolean] =
```

Comment:
The master and worker won't usually be on the same node, which means that a config could be enough for the master and worker components. These two configs can be merged as one. Maybe `celeborn.logConf` and print configs from the client could be OK.

Reply:
do we really need such a configuration?

how about 
```
if (log.debugEnabled) { // or trace level
  logDebug(Utils.redact(...))
}
```

Reply:
I think it would be good to have it as a part of the config, similar to `spark.logConf`.

Reply:
OK, let's use `celeborn.logConf.enabled`

Reply:
sounds good